docs/src/pages/components/Button/Button.example.tsx

```diff
@@ -5,201 +5,15 @@ import { Button } from "@cmsgov/design-system";
-          </td>
-        </tr>
-        <tr>
-          <td>
-            <Button className="ds-c-button--primary-outline">Button</Button>
-          </td>
-          <td>Secondary Teal outline</td>
-          <td>Primary Outline Button</td>
-          <td>
-            <code>className=&quot;ds-c-button--primary-outline&quot;</code>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            <Button className="ds-c-button--primary-alt-outline">Button</Button>
-          </td>
-          <td>Secondary Blue outline</td>
-          <td>Primary Alt Outline Button</td>
-          <td>
-            <code>className=&quot;ds-c-button--primary-alt-outline&quot;</code>
-          </td>
-        </tr>
-        <tr className="ds-base--inverse">
-          <td>
-            <Button>Button</Button>
-          </td>
-          <td>Secondary White</td>
-          <td>Default Button</td>
-          <td>none</td>
-        </tr>
-        <tr>
-          <td>
-            <Button disabled>Button</Button>
-          </td>
-          <td>Disabled</td>
-          <td>Any disabled button</td>
-          <td>
-            <code>disabled=true</code>
-          </td>
-        </tr>
-      </table>
-
-      <table cellSpacing={0}>
-        <thead>
-          <tr>
-            <th colSpan={4}>Link Buttons</th>
-          </tr>
-        </thead>
-        <tr>
-          <th></th>
-          <th>Sketch Name</th>
-          <th>Design System Name</th>
-          <th>Attributes</th>
-        </tr>
-        <tr>
-          <td>
-            <Button className="ds-c-button--primary ds-c-button--transparent">
-              Button
-            </Button>
-          </td>
-          <td>Teal link</td>
-          <td>Primary Transparent Button</td>
-          <td>
-            <code>
-              className=&quot;ds-c-button--primary
-              ds-c-button--transparent&quot;
-            </code>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            <Button className="ds-c-button--primary-alt ds-c-button--transparent">
-              Button
-            </Button>
-          </td>
-          <td>Blue link</td>
-          <td>Primary Alt Transparent Button</td>
-          <td>
-            <code>
-              className=&quot;ds-c-button--primary-alt
-              ds-c-button--transparent&quot;
-            </code>
-          </td>
-        </tr>
-        <tr className="ds-base--inverse">
-          <td>
-            <Button className="ds-c-button--inverse ds-c-button--transparent">
-              Button
-            </Button>
-          </td>
-          <td>Link on dark background</td>
-          <td>Inverse Transparent Button</td>
-          <td>
-            <code>
-              className=&quot;ds-c-button--inverse
-              ds-c-button--transparent&quot;
-            </code>
-          </td>
-        </tr>
-      </table>
+      <span className="ds-u-margin-right--1">
```

Comment:
@erinrep What do you think about a follow up PR to make the React and HTML examples identical? This way folks know what props to use to achieve a specific style?